src/main/java/frc/trigon/robot/utilities/JsonHandler.java

```diff
@@ -0,0 +1,71 @@
+package frc.trigon.robot.utilities;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import edu.wpi.first.wpilibj.Filesystem;
+
+import java.io.*;
+
+public class JsonHandler {
+    private static final Gson gson = new GsonBuilder().setPrettyPrinting().create();
+    private static final String path = Filesystem.getDeployDirectory().getPath() + "\\";
+
+    /**
+     * Parses the given object to a new or existing JSON file,
```

Comment:
You don't parse it to a file, you parse it to JSON. You then write the JSON to a new or existing file.